<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlaxaFlow Desktop</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* --- ESTILO GLOBAL CLEAN ALAXA --- */
QMainWindow {
    background-color: #FAFAFA;
}
QWidget {
    font-family: 'Segoe UI', sans-serif;
    font-size: 14px;
    color: #3E2723;
}

/* --- HEADER --- */
QFrame#TopBar {
    background-color: #4E342E; 
    border-bottom: 3px solid #FFB74D;
}
QLabel#HeaderTitle {
    color: #FFFFFF;
    font-weight: bold;
    font-size: 20px;
}
QPushButton#LogoutBtn {
    background-color: transparent;
    color: #FFB74D;
    border: 1px solid #FFB74D;
    border-radius: 4px;
    padding: 5px 15px;
}
QPushButton#LogoutBtn:hover {
    background-color: #5D4037;
    color: #FFFFFF;
}

/* --- COLUMNAS --- */
QFrame#col_pendiente, QFrame#col_proceso, QFrame#col_revision, QFrame#col_finalizado {
    background-color: #EFF1F3;
    border-radius: 8px;
    border: 1px solid #D7CCC8;
}
QLabel {
    font-weight: bold;
    font-size: 16px;
    color: #4E342E;
    padding: 5px;
}

/* --- SCROLL AREA (Transparente) --- */
QScrollArea {
    border: none;
    background: transparent;
}
QWidget#scrollContents_pendiente, QWidget#scrollContents_proceso, 
QWidget#scrollContents_revision, QWidget#scrollContents_finalizado {
    background: transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="TopBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="HeaderTitle">
         <property name="text">
          <string>AlaxaFlow</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnLogout">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Cerrar Sesión</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentArea" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="col_pendiente">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_1">
            <property name="text">
             <string>PENDIENTE</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_1">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollContents_pendiente">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>271</width>
               <height>638</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="layout_pendiente">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="col_proceso">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>EN PROCESO</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollContents_proceso">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>271</width>
               <height>638</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="layout_proceso">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="col_revision">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>REVISIÓN</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_3">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollContents_revision">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>271</width>
               <height>638</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="layout_revision">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="col_finalizado">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>FINALIZADO</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_4">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollContents_finalizado">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>271</width>
               <height>638</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="layout_finalizado">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
